Wave/Wave/SingleSelectionSetting.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  SingleSelectionSetting.swift
+//  Wave
+//
+//  Created by Jimmy McDermott on 7/5/16.
+//  Copyright © 2016 BlueLine Labs. All rights reserved.
+//
+
+import Foundation
+
+public final class SingleSelectionSection: SectionType, SettingType {
+    
+    public var possibleValues: [Any]
+    public var enableCustom = false
+    public var customOption: RowType?
+    public var name: String
+    private var settings = [StringSetting]()
+    
+    public init(possibleValues: [Any], enableCustom: Bool, customOption: RowType?, name: String) {
```

Comment:
Yeah, I'm not sure what the best way to approach this is. Ideally we would have had this be a generic class, but we all know how that turned out. Second best option might just be to have `StringSingleSelectionSection` and `IntSingleSelectionSection`?